### ISSUE TITLE
Add wiki to Freecad documentation

### ISSUE THREAD

**Issue by hasecilu** [APEbbers/FreeCAD-Ribbon#78]
![image](https://github.com/user-attachments/assets/b4908b98-7389-4f34-90fa-c71ea82849a3)

The page `file:///wiki` is opened on the browser which is not helpful =/.

I can create pages on the FreeCAD Wiki so if you want I can create the `Ribbon_Workbench` page or is https://wiki.freecad.org/ModernUI_Workbench/en enough? For the moment I'm new with this addon (and prior versions) so I lack the knowledge to write all documentation.

We can open documentation inside FreeCAD with:

```python
import Help
Help.show("PieMenu Workbench")
```

In my case works with FreeCAD with Qt6, but on FreeCAD weekly with Qt5 I got (Arch Linux btw):

>    Could not find QtWebEngineProcess
>    /tmp/.mount_freecaltRcxz/AppRun: line 41: 246119 Aborted (core dumped) "${MAIN}" "$@"

---

Also, consider adding tooltips to buttons that moves commands, maybe the icons of #62 can be used here.

![image](https://github.com/user-attachments/assets/83b47f7d-a8f5-49ad-aa63-49c7f2b8dafc)

**Comment by Reqrefusion** [comment]
https://wiki.freecad.org/User:Reqrefusion/FreeCAD-Ribbon/

I started by moving the existing documentation to the wiki, but of course the standards are different in the wiki.

**Comment by APEbbers** [comment]
Hi,

Is there a difference between markdown and the wiki perhaps? If so, maybe there is a conversion tool. function of some kind?

**Comment by Reqrefusion** [comment]
Yes, wikis use their own markup language, wikitext. Pandoc is generally used, but there are plenty of sample scripts online.
